mzLib/FlashLFQ/Interfaces/IQuantifiablePeptide.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+
+namespace FlashLFQ
+{
+    /// <summary>
+    /// Defines the interface for peptides that can be quantified
+    /// </summary>
+    public interface IQuantifiablePeptide
+    {
+        /// <summary>
+        /// The peptide sequence
+        /// </summary>
+        string Sequence { get; }
```

Comment:
We should probably use BaseSequence or FullSequence so it maps better with our existing objects.